lib/api.js

```diff
@@ -85,7 +85,7 @@ function apiCall(httpMethod, options, callback) {
         if (err) {
           log(err, { meta });
 
-          callback(err);
+          callback(err, null, null, req);
```

Comment:
Better to be more specific and use `undefined`, just in case the callback starts doing something with `null` values.
```suggestion
          callback(err, undefined, undefined, req);
```
See https://codeburst.io/javascript-null-vs-undefined-20f955215a2

Reply:
OMG null is an object 😱